config/routes.rb

```diff
@@ -20,13 +20,15 @@
     concerns :exportable
   end
 
-  resources :bookmarks do
-    concerns :exportable
-
-    collection do
-      delete 'clear'
-    end
-  end
+  
```

Comment:
is there a route to remove for search history?

Reply:
That comes from blacklight and is still active in the current repo so I assumed that it was ok to keep it here as well.